package-lock.json

```
@@ -1,8 +1,3122 @@
 {
   "name": "stoa",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

```suggestion
  "lockfileVersion": 1,
```
Can you take it back?
Which npm version are you using?

---

I am using npm v7.14.0

---

Which npm version should I use?